tests/test_models.py

```diff
@@ -1345,17 +1346,19 @@ def test_ankle_draft():
         ankle_draft(25, 25, 0.3, 50, 1.2, 0.5, 7)
 
 
-def test_utci():
-    data_test_adaptive_ashrae = (
-        [  # I have commented the lines of code that don't pass the test
-            {"tdb": 25, "tr": 27, "rh": 50, "v": 1, "return": {"utci": 25.2}},
-            {"tdb": 19, "tr": 24, "rh": 50, "v": 1, "return": {"utci": 20.0}},
-            {"tdb": 19, "tr": 14, "rh": 50, "v": 1, "return": {"utci": 16.8}},
-            {"tdb": 27, "tr": 22, "rh": 50, "v": 1, "return": {"utci": 25.5}},
-            {"tdb": 27, "tr": 22, "rh": 50, "v": 10, "return": {"utci": 20.0}},
-            {"tdb": 27, "tr": 22, "rh": 50, "v": 16, "return": {"utci": 15.8}},
-        ]
-    )
+@pytest.fixture
+def data_test_adaptive_ashrae():
```

Comment:
could you please change this from `data_test_adaptive_ashrae` to `data_test_utci`. I originally erroneously named this variable.